### PR TITLE
MNTOR-2035: add content to data broker tooltip

### PR DESCRIPTION
**Pull request by mansaj**
# References: 
Jira: MNTOR-2035

# Description
Add disclaimer for data broker tooltip

# Screenshot (if applicable)
Screenshot 2023-09-11 at 12 09 31 PM

# How to test

# Checklist (Definition of Done)
- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: locales-pending/fix.ftl [hunk 1]
```diff
@@ -30,7 +30,7 @@ fix-flow-data-broker-profiles-view-data-broker-profiles-view-info-on-sites = Vie
 fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-trigger-label = More info
 fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-title = About these profiles
 fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph1 = Data brokers make up a $240 billion industry of gathering your personal info and selling it to anyone searching for you. They scrape your data from publicly available sources to create profiles that include your name(s), current and previous addresses, family member names, criminal history, and much more.
-fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph2 = We found these profiles by searching 190 data broker sites for the name, location, and date of birth you provided. These profiles are currently live and available to anyone searching for you.
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph2 = We found these profiles by searching 190 data broker sites for the name, location, and date of birth you provided. These profiles are currently live and available to anyone searching for you. { -brand-Mozilla } is not associated with these data broker sites.
```
[the remark this answers is not in the document]
Comment (mansaj) [reply locales-pending/fix.ftl]: Fixed thanks

**Review by flozia — APPROVED** [review]
Looks good!

File: locales-pending/fix.ftl [hunk 2]
```diff
@@ -30,7 +30,13 @@ fix-flow-data-broker-profiles-view-data-broker-profiles-view-info-on-sites = Vie
 fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-trigger-label = More info
 fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-title = About these profiles
 fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph1 = Data brokers make up a $240 billion industry of gathering your personal info and selling it to anyone searching for you. They scrape your data from publicly available sources to create profiles that include your name(s), current and previous addresses, family member names, criminal history, and much more.
-fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph2 = We found these profiles by searching 190 data broker sites for the name, location, and date of birth you provided. These profiles are currently live and available to anyone searching for you.
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan.
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph2 =
```
Comment (flozia) [inline locales-pending/fix.ftl:35]: Just as a note: I think we can always expect more than `1` broker site here and other strings that are taking the total data broker count not using a variable string. Not sure if we are unnecessarily creating more work for localizers @flodolo?

Comment (mansaj) [reply locales-pending/fix.ftl:35]: Yea, I kind of agree with this.. However, there are quite a few places in the existing fluent files where we have this 1/1+ conditions for this exact variable.. so

Comment (flodolo) [reply locales-pending/fix.ftl:35]: Most locales have complex plural forms, ">1" doesn't mean that they have just one form.
https://www.unicode.org/cldr/charts/43/supplemental/language_plural_rules.html

Comment (mansaj) [reply locales-pending/fix.ftl:35]: So you'd suggest we keep the conditionals? @flodolo

**Review by flodolo — APPROVED** [review]

File: locales-pending/fix.ftl [hunk 3]
```diff
@@ -30,7 +30,13 @@ fix-flow-data-broker-profiles-view-data-broker-profiles-view-info-on-sites = Vie
...
-fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph2 = We found these profiles by searching 190 data broker sites for the name, location, and date of birth you provided. These profiles are currently live and available to anyone searching for you.
+# Variables:
+# $data_broker_sites_total_num is the total number of data broker sites available to scan.
+fix-flow-data-broker-profiles-view-data-broker-profiles-more-dialog-paragraph2 = 
+  { $data_broker_sites_total_num ->
+    [one] We found these profiles by searching { $data_broker_sites_total_num } data broker site for the name, location, and date of birth you provided. These profiles are currently live and available to anyone searching for you. { -brand-mozilla } is not associated with these data broker sites.
+    *[other] We found these profiles by searching { $data_broker_sites_total_num } data broker sites for the name, location, and date of birth you provided. These profiles are currently live and available to anyone searching for you. { -brand-Mozilla } is not associated with these data broker sites.
```
Comment (flodolo) [inline locales-pending/fix.ftl:38]:
```suggestion
    *[other] We found these profiles by searching { $data_broker_sites_total_num } data broker sites for the name, location, and date of birth you provided. These profiles are currently live and available to anyone searching for you. { -brand-mozilla } is not associated with these data broker sites.
```

To be clear, you also need to update it here.

**Pull request merged**